ui/app/AppLayouts/Wallet/components/TokenSettingsModalContent.qml

```diff
@@ -8,9 +8,8 @@ import "../../../../shared/status"
 import "../../Chat/ContactsColumn"
 import "../data/"
 
-Item {
+Rectangle {
```

Comment:
Why change it to a Rectangle? If you want to keep it as a Rectangle, you need to change the `color`, because it defaults to `white` otherwise

Reply:
Yeap, that's a left over. Changing it back